### DISCUSSION TITLE
MP3 issue - not playing in audio player. Possible large file size problem?

### DISCUSSION THREAD

**Discussion by kris-joseph** [CollectionBuilder/collectionbuilder.github.io#105] [Q&A] [answered]
I'm seeing an issue with CollectionBuilder sites not playing audio files properly. 

I'm led to believe that GitHub Pages is okay with large files up to about 50MB, and I've got some people working with audio files that are 20MB or less.  Unlike the demo sites, which [play audio correctly](https://kris-joseph.github.io/cb_4840/item.html?id=cbdemo003), these sites do not respond when the audio player is used, and the "Download MP3" button gives an invalid MIME type error in Firefox (a copy of a site I'm troubleshooting has [an audio item here](https://kris-joseph.github.io/ParsonCopy/item.html?id=coll005) that demonstrates the issue). Even if I trim the audio to a file about 1MB in size, I have the same problem. The [full repo of my troubleshooting site is here](https://github.com/kris-joseph/ParsonCopy).

I've been poring over the metadata and other elements and cannot see where the issue is. Has anyone else seen this, and can you spot what's gone wrong? Thanks!

**Comment by dcnb** [comment, marked as answer]
Hi @kris-joseph, this is super weird! 

I wonder if it has something to do with your repository being set up for Git LFS. When I tried to pull down your repository I got the following attached error saying Git LFS (Large File Storage) was initialized. 

<img width="404" alt="Screenshot 2023-11-20 at 3 14 36 PM" src="https://github.com/CollectionBuilder/collectionbuilder.github.io/assets/10931737/fb30398f-39e9-4b64-8173-f4f73714eac2">

 I wanted to make sure everything worked, so I just made a copy of GH and replaced the data quickly with yours. It worked fine: 

https://dcnb.github.io/testaudio/item.html?id=coll005

(I will delete this when you reply!)

So I think you just need to either start a new repository and move your data over, or get rid of Git LFS on that repository. The files aren't that large, so they shouldn't need any help playing. You can also turn off Git LFS for the repo, but it seems like you might want to delete it and restart. 

You didn't mention anything about turning on LFS though, so I don't know if it's just something that got weirdly triggered or a setting you have more broadly set up? I am no expert on LFS. Looks like you can check how/where you might be using it by looking at your billing page: https://github.com/settings/billing/summary

Overall though, this is a unique error that I haven't seen before. I'm hoping the old delete and retry with a clean repository will work for you, but please let us know if there are more issues!

@dcnb

**Comment by kris-joseph** [comment]
By gosh, I think you nailed it! At the end of the day, I'm not sure how LFS ended up enabled by default with a clean install of GitHub Desktop, but I'll; sort that out on my end. I have been able to verify that removing the audio file from LFS results in a working item page, which pretty much takes care of the problem!

Thanks very much for the quick response -- I likely would not have been able to sort this out (and was digging through source code to figure out how to link to the media.github.com site with Liquid... it was getting ugly....!)

**Comment by dcnb** [comment]
Awesome! I'll go delete that repository now
